docs/pages/docs/05-distributed-analytics/compute-centrality.md

```diff
@@ -0,0 +1,117 @@
+More importantly, how do we measure importance in a graph?
+
+Indicators of centrality identify the most important vertices within a graph. 
+[Wikipedia](https://en.wikipedia.org/wiki/Centrality)
+
+Currently, Graql supports two algorithms for computing centrality: 
+- Degree 
+- K-Core
+
+## Degree
+
+The degree of an instance gives the number of other instances directly connected to the instance. 
+For an entity, such asa `person`, this means the number of connections, 
+for example instances of a `marriage` relationship.
+The `person` with the highest degree, or the greatest number of marriages, is arguably the most "interesting".
+
+In analyics, the `degree` algorithm computes how many arrows (edges) there are attached to instances in the graph. 
+A map is returned containing degree and instances with the degree (map<key=degree, value=set of instance>). If we call:
+
+```graql
+compute centrality; using degree;
+```
+
+On the graph below we expect the degrees of the people to be:
+
+* **Barbara Herchelroth**: 3 - two marriages and a child
+* **Jacob Young**: 2 - a marriage and a child
+* **Mary Young**: 2 - two parents
+* **John Newman**: 1 - a marriage
+
+We do not count the edges labelled isa, because they are connected to the types and are not considered by analytics.
+Don't forget that the relationships themselves will also have degrees.
+
+{% include note.html content="Don't forget that the relationships themselves will also have degrees." %}
+
+![A simple social network.](/images/analytics_degree_full.png)
+
+## K-Core (Coreness)
+
+Coreness is a measure that can help identify tightly interlinked groups within a network.
+An instance have coreness k if the instance belongs to a 
+[k-core](https://en.wikipedia.org/wiki/Degeneracy_(graph_theory)#k-Cores) but not to any
+(k+1)-core.
+
+We can compute the coreness centrality using the following:
+
+```graql
+compute centrality; using k-core;
+```
+
+Similar to degree, a map is returned.
+
+{% include note.html content="Relationship instances will NOT have coreness." %}
+
+Additionally, if we only care about the entities that have higher coreness, we can set the minimum value of k,
+using the following command:
+
+```graql
+compute centrality; using k-core where min-k = 10;
+```
+
+So the result map will only include entities with coreness greater than or equal to 10. 
+
+{% include note.html content="k value can not be smaller than 2" %}
+
+## Centrality within a subgraph
+
+Consider that in this graph, people with more marriages are more interesting.
+We can use the subgraph functionality to restrict the graph to only see people and who they are married to.
+Once the graph has been restricted we can determine the number of marriages by computing the degree:
+
+```graql
+compute centrality in person, marriage; using degree;
+```
+
+The result will now be:
+
+* **Barbara Herchelroth**: 2
+* **Jacob Young**: 1
+* **Mary Young**: 0
+* **John Newman**: 1
```

Comment:
In this example, none of the relationships have degree because they are not role players. So the result doesn't contain relationships. I'll try to explain this in `note` section.